### PR TITLE
Error handling

### PR DESCRIPTION
**Pull request by zakorgy**

### CODE REVIEW

**Comment by mmatyas** [comment]
lgtm

File: components/script/dom/bluetoothremotegattcharacteristic.rs [hunk 1]
```diff
@@ -161,10 +158,9 @@ impl BluetoothRemoteGATTCharacteristicMethods for BluetoothRemoteGATTCharacteris
             BluetoothObjectMsg::BluetoothWriteValue => (),
             BluetoothObjectMsg::Error {
                 error
-            } => {
-                println!("{}", error);
-            },
+            } => return Err(Type(error)),
             _ => unreachable!()
         };
+        Ok(())
```
Comment (dati91) [inline components/script/dom/bluetoothremotegattcharacteristic.rs]: move this up to `BluetoothObjectMsg::BluetoothWriteValue => Ok(()),` and remove `;` from line167 so the `return` won't be needed

File: components/script/dom/bluetoothremotegattdescriptor.rs [hunk 2]
```diff
@@ -124,10 +124,9 @@ impl BluetoothRemoteGATTDescriptorMethods for BluetoothRemoteGATTDescriptor {
             BluetoothObjectMsg::BluetoothWriteValue => (),
             BluetoothObjectMsg::Error {
                 error
-            } => {
-                println!("{}", error);
-            },
+            } => return Err(Type(error)),
             _ => unreachable!()
         };
+        Ok(())
```
Comment (dati91) [inline components/script/dom/bluetoothremotegattdescriptor.rs]: same as above

File: components/script/dom/bluetoothremotegattserver.rs [hunk 3]
```diff
@@ -74,15 +77,15 @@ impl BluetoothRemoteGATTServerMethods for BluetoothRemoteGATTServer {
             BluetoothObjectMsg::Error {
                 error
             } => {
-                println!("{}", error);
+                return Err(Type(error));
             },
             _ => unreachable!()
         }
-        Root::from_ref(self)
+        Ok(Root::from_ref(self))
```
Comment (dati91) [inline components/script/dom/bluetoothremotegattserver.rs]: you can refactor this as well as mentioned above

**Comment by dati91** [comment]
lgtm

**Pull request merged**